lib/govuk_personalisation/urls.rb

```diff
@@ -15,11 +15,11 @@ def self.sign_out
     find_govuk_url(var: "SIGN_OUT", application: "frontend", path: "/sign-out")
   end
 
-  # Find the GOV.UK URL for the "your account" page
+  # Find the external URL for the "your account" page
   #
   # @return [String] the URL
   def self.your_account
-    find_govuk_url(var: "YOUR_ACCOUNT", application: "frontend", path: "/account/home")
+    find_external_url(var: "YOUR_ACCOUNT", url: "https://#{digital_identity_domain}")
```

Comment:
Oh cool, so that comes from here:
https://github.com/alphagov/govuk_personalisation/blob/67ba5235e6b70495aba7a19b90c9a5123fa93251/lib/govuk_personalisation/urls.rb#L91

🤔 
There's no fallback for this, right?

So we're changing this from `"/account/home"` to whatever `DIGITAL_IDENTITY_ENVIRONMENT` is...

I can't see that in puppet right now... so don't we need to deploy a `DIGITAL_IDENTITY_ENVIRONMENT` which is set to `"/account/home"`, else this will just be nil?

Reply:
I'm guesing I just missed a definition of `DIGITAL_IDENTITY_ENVIRONMENT` somewhere?

Reply:
Looks like that's what we're doing over there:
https://github.com/alphagov/govuk-puppet/pull/11728/commits/c46c36b5fa3e12d7af8c995f487c4f9af8883ac3

But that uses `GOVUK_PERSONALISATION_YOUR_ACCOUNT_URI` and not `DIGITAL_IDENTITY_ENVIRONMENT`

Am I missing something?

Reply:
Doh. It's because we're using `find_external_url` not `find_govuk_url`...

and _that_ is a fetch with a fallback.
Nice. That makes sense!

Phew got there in the end 😂

Reply:
Haha! Sorry, I maybe should have linked to [my Jira comment](https://govukverify.atlassian.net/browse/GUA-72?focusedCommentId=34874) where I explained the plan in a bit more detail